tetris.py

```diff
@@ -337,21 +337,29 @@ def __init__(self, main_board, topbar_time):
         self.paused_time = 0
         self.game_status = GameStatus.in_progress
         self.timer()
+        self.no_retry_pause = False
```

Comment:
The logic seems to be confusingly backwards here:

```suggestion
        retry_pause = True
```